source/server-discovery-and-monitoring/server-monitoring.rst

```diff
@@ -78,34 +76,40 @@ set tags. In SDAM terms, a significant topology change on the server means the
 client's ServerDescription is out of date. Standalones and mongos do not
 currently experience significant topology changes but they may in the future.
 
-regular isMaster command
-````````````````````````
+regular hello or legacy hello command
+`````````````````````````````````````
 
-A default ``{isMaster: 1}`` command where the server responds immediately.
+A default ``{hello: 1}`` or legacy hello command where the server responds immediately.
 
 
-streamable isMaster command
-```````````````````````````
+streamable hello or legacy hello command
+````````````````````````````````````````
 
-The isMaster command feature which allows the server to stream multiple
+The hello or legacy hello command feature which allows the server to stream multiple
 replies back to the client.
 
 RTT
 ```
 
-Round trip time. The client's measurement of the duration of one isMaster call.
+Round trip time. The client's measurement of the duration of one hello or legacy hello call.
 The RTT is used to support `localThresholdMS`_ from the Server Selection spec.
 
 
 Monitoring
 ''''''''''
 
-The client monitors servers using the isMaster command. In MongoDB 4.4+, a
-monitor uses the `Streaming Protocol`_ to continuously stream isMaster
+The client monitors servers using the hello or legacy hello commands. In MongoDB 4.4+, a
+monitor uses the `Streaming Protocol`_ to continuously stream hello or legacy hello
 responses from the server. In MongoDB <= 4.2, a monitor uses the
 `Polling Protocol`_ pausing heartbeatFrequencyMS between `checks`_.
 Clients check servers sooner in response to certain events.
 
+If a `server API version`_ is requested, then the driver must use hello for monitoring.
+If a server API version is not requested, the initial handshake using the legacy hello
+command must include `helloOk: true`. If the response contains `helloOk: true`, then the
+driver must use the `hello` command for monitoring. If the response does not contain
+`helloOk: true`, then the driver must use the legacy hello command for monitoring.
```

Comment:
Nice, does this section cover the spec changes for DRIVERS-1750?

Reply:
Yes, this is the spec change to go along with DRIVERS-1750.